tensorflow/lite/arena_planner_test.cc

```diff
@@ -210,6 +210,19 @@ TEST_F(ArenaPlannerTest, EmptyGraph) {
   Execute(0, 10);
 }
 
+TEST_F(ArenaPlannerTest, DeallocationOfInputTensor) {
+  // This is a negative TC, which will try to make sure that no allocation for
+  // input tensors is done, hence making it call
+  // CalculateDeallocationOfInternalTensors with negative node_index, since
+  // previous check was doing comparison of node_index which was int and
+  // unsigned int, implicit conversion was passing this case, as the negative
+  // number was converted to unsigned it making it random big int.The new check
```

Comment:
"was converted to unsigned, making it invalid. The new check..."

Reply:
Sure, same is updated in the code.

Regards
Amit